src/audio/module_adapter/module_adapter.c

```diff
@@ -1578,10 +1578,12 @@ int module_adapter_reset(struct comp_dev *dev)
 	}
 #endif /* CONFIG_ZEPHYR_DP_SCHEDULER */
 	if (IS_PROCESSING_MODE_SINK_SOURCE(mod)) {
+		int i;
```

Comment:
I wonder if we could build testbench in C99 mode. When we build SOF currently (at least Zephyr builds), we are using C99 mode:
```
-std=c99 -std=gnu99
```

... so basicly C99'ism can pass through our CI checks and testbench will always be in catch-up mode. Zephyr is C99+ -> https://docs.zephyrproject.org/latest/develop/languages/c/index.html

That's a harder question where to set this, but would be a longer term solution.

Reply:
Need to check how to add this, this commit could be possible to drop with that change.